src/test/java/Model/CarTest.java

```diff
@@ -0,0 +1,48 @@
+package Model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class CarTest {
+    private Car car;
+
+    @BeforeEach
+    void setUp() {
+        car = new Car(0);
+    }
+
+    @DisplayName("전진할 수 있는지 조건을 체크한다.")
+    @ParameterizedTest
+    @CsvSource({
+            "1, false",
+            "2, false",
+            "4, true",
+            "5, true",
+            "6, true",
+            "9, true"
+    })
+    public void moveForwardCondition(int randomNumber, boolean expected) {
+        assertThat(car.canMoveForward(randomNumber)).isEqualTo(expected);
+    }
+
+    @DisplayName("난수에 따라 자동차는 전진 혹은 정지한다.")
+    @ParameterizedTest
+    @CsvSource({
+            "1, 0",
+            "2, 0",
+            "3, 0",
+            "4, 1",
+            "5, 1",
+            "9, 1"
```

Comment:
```suggestion
            "0, 0",
            "3, 0",
            "4, 1",
            "9, 1"
```
테스트 데이터가 무조건 많은 것이 좋은 것은 아님
경계값 위주로 최소한의 테스트 데이터로 같은 효과를 낼 것을 추천